angular/projects/spark-angular/src/lib/directives/sprk-link/sprk-link.stories.ts

```diff
@@ -81,37 +87,77 @@ export const simple = () => ({
 
 simple.story = {
   parameters: {
-    jest: [
-      'sprk-link.directive',
-    ],
+    jest: ['sprk-link.directive'],
+  },
+};
+
+export const light = () => ({
+  moduleMetadata: modules,
+  template: `
+    <div sprkBox [ngStyle]="{'background-color':'#1c1b1a'}">
```

Comment:
Can you set the `decorators` property on this story individually and apply the sprkBox here instead of doing it the way you have? That away, when they go to copy the code it doesn't include the Box `div`.